src/features/utils/feature-base.js

```diff
@@ -0,0 +1,48 @@
+import { getInfo, isConfigured, getRuntime } from '../../common/config/config'
+import { ee } from '../../common/event-emitter/contextual-ee'
+import { configure } from '../../loaders/configure/configure'
+import { gosCDN } from '../../common/window/nreum'
+
+export class FeatureBase {
+  constructor(agentIdentifier, aggregator, featureName) {
+    /** @type {string} */
+    this.agentIdentifier = agentIdentifier
+    /** @type {Aggregator} */
+    this.aggregator = aggregator
+    /** @type {ContextualEE} */
+    this.ee = ee.get(agentIdentifier, getRuntime(this.agentIdentifier).isolatedBacklog)
```

Comment:
(Btw, ee.get doesn't take a 2nd arg.. at least not yet.)

Reply:
remnant, can be removed in future ticket

Reply:
https://issues.newrelic.com/browse/NEWRELIC-6480